From 9ce9ddb4a84fb467602b716575ea1d8f2bab0c39 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Wed, 7 Dec 2022 15:00:35 +0100
Subject: [PATCH] ucm2: HDA: Update Mic LED settings for ACP DMIC

Users expect to turn the LED on when only internal mic is off, but
it makes sense to turn this LED when all internal inputs are off.

This configuration may be changed when the affected applications
are updated.

BugLink: https://bugzilla.redhat.com/show_bug.cgi?id=2134824
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/HDA/HDA.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ucm2/HDA/HDA.conf b/ucm2/HDA/HDA.conf
index 64ffe83..47dfc05 100644
--- a/ucm2/HDA/HDA.conf
+++ b/ucm2/HDA/HDA.conf
@@ -55,6 +55,7 @@ If.use {
 					cset-new "name='Mic ACP LED Capture Switch' type=bool,count=1 off"
 					exec "-/sbin/modprobe snd_ctl_led"
 					sysw "-/class/sound/ctl-led/mic/card${CardNumber}/attach:Mic ACP LED Capture Switch"
+					sysw "-/class/sound/ctl-led/mic/card${CardNumber}/detach:Capture Switch"
 				]
 			}
 		}
-- 
2.34.1

